zebra-state/src/service/tests.rs

```diff
@@ -312,6 +312,15 @@ proptest! {
         for block in finalized_blocks {
             let expected_block = block.clone();
 
+            let expected_action;
+            if expected_block.height <= block::Height(1) {
+                // 0: reset by both initialization and the Genesis network upgrade
+                // 1: reset by the BeforeOverwinter network upgrade
+                expected_action = TipAction::reset_with(expected_block.clone().into());
+            } else {
+                expected_action = TipAction::grow_with(expected_block.clone().into());
+            }
```

Comment:
Nit: Style suggestion to a bit more concise:

```suggestion
            let expected_action = if expected_block.height <= block::Height(1) {
                // 0: reset by both initialization and the Genesis network upgrade
                // 1: reset by the BeforeOverwinter network upgrade
                TipAction::reset_with(expected_block.clone().into())
            } else {
                TipAction::grow_with(expected_block.clone().into())
            };
```

Reply:
Ok, I'll go do this for the code below as well.